keyboards/bastardkb/charybdis/readme.md

```diff
@@ -18,32 +18,61 @@ Check out the [keyboard build guides](https://docs.bastardkb.com) for the Charyb
 
 ## Building the firmware
 
+You must specify the shield version when compiling/flashing the firmware. 
+
 ### Charybdis (4x6)
 
 The `default` keymap is inspired from the original [Dactyl Manuform](../../handwired/dactyl_manuform) layout.
 
+* For Elite-C v1 shield
+
 ```shell
-qmk compile -kb bastardkb/charybdis/4x6 -km default
+qmk compile -kb bastardkb/charybdis/4x6/1.x/elitec -km default
+```
```

Comment:
This needs to be indented to be part of the bullet point I believe.

Reply:
good point, updated